dev/MRTCore/mrt/Core/unittests/MrmTests.cpp

```diff
@@ -343,58 +370,58 @@ TEST_CLASS(BasicTest)
     TEST_METHOD(EnumerateResourceMap)
     {
         MrmManagerHandle resourceManager;
-        Assert::AreEqual(MrmCreateResourceManager(L".\\resources.pri", &resourceManager), S_OK);
+        VERIFY_ARE_EQUAL(MrmCreateResourceManager(L".\\resources.pri", &resourceManager), S_OK);
 
         // Enumerate under Microsoft.UI.Xaml/Resources
         MrmMapHandle childResourceMap;
-        Assert::AreEqual(MrmGetChildResourceMap(resourceManager, nullptr, L"Microsoft.UI.Xaml", &childResourceMap), S_OK);
+        VERIFY_ARE_EQUAL(MrmGetChildResourceMap(resourceManager, nullptr, L"Microsoft.UI.Xaml", &childResourceMap), S_OK);
 
         MrmMapHandle childChildResourceMap;
-        Assert::AreEqual(MrmGetChildResourceMap(resourceManager, childResourceMap, L"Resources", &childChildResourceMap), S_OK);
+        VERIFY_ARE_EQUAL(MrmGetChildResourceMap(resourceManager, childResourceMap, L"Resources", &childChildResourceMap), S_OK);
 
         UINT32 count;
-        Assert::AreEqual(MrmGetResourceCount(resourceManager, childChildResourceMap, &count), S_OK);
-        Assert::AreEqual<UINT32>(count, 78);
+        VERIFY_ARE_EQUAL(MrmGetResourceCount(resourceManager, childChildResourceMap, &count), S_OK);
+        VERIFY_ARE_EQUAL(count, static_cast<UINT32>(78));
```

Comment:
NIT: You should be able to avoid the cast by doing 78u.

(due to the way the verify macros use templates the implicit conversion of a constant to an unsigned integer is lost; but you can fix that by explicitly marking the constant as unsigned)

Reply:
Thanks! Good idea to use 78u instead of a cast.